docs/development.md

```diff
@@ -99,7 +103,21 @@ The dev version of the manifests can be generated with
 
 1. [Install the gcloud cli][gcloud_sdk].
 2. Set project: `gcloud config set project YOUR_PROJECT_NAME`.
-3. Build & push dev images: `make docker-push-dev`.
+3. To build images with custom tags, edit [.bazelrc](./.bazelrc) as follows:
```

Comment:
Should we prefix optional steps with `(Optional)` or something similar to communicate these aren't required for the workflow that follows the preset conventions?

Reply:
I think we should always recommend folks to use custom tags so that they don't accidentally end up pushing wrong tags resulting in a prod deployment. Ideally, you would want to use the image SHA in all your deployments. But that's a different topic :)